_rules/button-accessible-name-97a4e1.md

```diff
@@ -21,10 +21,12 @@ acknowledgements:
 
 ## Applicability
 
-The rule applies to elements that are [included in the accessibility tree][] and have a [semantic role](#semantic-role) of `button`, except for `input` elements whose `type` attribute is in the [`Image Button` state](<https://html.spec.whatwg.org/multipage/input.html#image-button-state-(type=image)>).
+The rule applies to elements that are [included in the accessibility tree][] and have a [semantic role](#semantic-role) of `button`, including `input` elements whose `type` attribute is in the [`Image Button` state](<https://html.spec.whatwg.org/#image-button-state-(type=image)>).
 
 **Note:** `input` elements have a `type` attribute in the `Image button` state if it is set to any case-insensitive match of `image` (most of the time, using `<input type="image">`).
 
+**Note:** The specification of the [`type`](https://html.spec.whatwg.org/#states-of-the-type-attribute) attribute describes in detail how to map the value of the attribute to its corresponding state.
+
```

Comment:
I think both these notes can now go away.

Reply:
Are either of these notes required later in regard a specific example. Required for understanding, as opposed to nice-to-have.

Reply:
If they are only required for understanding an example or two, they should probably be moved to the description of these examples. With the new applicability which has nothing to do with `type` or `Image button`, they look completely out of the place and it is not clear why a rule about buttons need this (for example, why precising how to put an `input` in the `Image button` state, but not precise how to put it in the `Button`, `Reset Button` or `Submit Button` state? And why even mention `input` elements on a rule about buttons?)